temporal_sqlalchemy/bases.py

```diff
@@ -21,9 +23,9 @@
 
 
 class EntityClock(object):
-    tick = sa.Column(sa.Integer, primary_key=True, autoincrement=False)
-    timestamp = sa.Column(sa.DateTime(True),
-                          server_default=sa.func.current_timestamp())
+    id = sa.Column(sap.UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
+    tick = sa.Column(sa.Integer, nullable=False)
+    timestamp = sa.Column(sa.DateTime(True), server_default=sa.func.current_timestamp())
```

Comment:
Do we need to explicitly set nullable=False here too? I see we did it in the migration, but I'm not sure how sqlalchemy treats this.